src/api/api.ts

```diff
@@ -8,4 +8,36 @@ const fetcher: AxiosInstance = axios.create({
   },
 });
 
+function isPublicRequest(method?: string, url?: string): boolean {
+  if (!method || !url) return false;
+  if (url.startsWith('/oauth')) {
+    return true;
+  }
+  if (url === '/auth/login') {
+    return true;
+  }
+  if (url === '/users') {
+    return true;
+  }
+  if (method === 'get' && url.startsWith('/my-activities')) {
+    return true;
+  }
```

Comment:
```suggestion
  if (method === 'get' && url.startsWith('/activities')) {
    return true;
  }
```
❗ public 한건 my-activities가 아니라 activities 같아요!